File: conditionals/time_of_day.py
#samantha naranjo, time of day py

hour = int(input("What hour is it? (0-23): "))  

if hour < 12:  
    print("good morning!")  
elif hour >= 12 and hour < 18:  
    print("good afternoon!")  
else:  
    print("good evening!")  

File: conditionals/time_of_day.c
//samantha naranjo, time of day c
#include <stdio.h>  
  
int main() {  
    int hour;  
    printf("What hour is it? (0-23): ");  
    scanf("%d", &hour);  
      
    if (hour < 12) {  
        printf("Good morning!\n");  
    } else if (hour >= 12 && hour < 18) {  
        printf("Good afternoon!\n");  
    } else {  
        printf("Good evening!\n");  
    }  
    
}  
